### ISSUE TITLE
Change default skills sampling size during datamixing to "1.0" from "30"

### ISSUE THREAD

**Issue by bbrowning** [instructlab/sdg#420]
We had a hardcoded sampling size of 30 samples per leaf node of custom skills. That's a very specific number that may be appropriate in some cases, but we're seeing more commonly users having only 1 or a few skill nodes and wanting more of their generated skill data in the mixed output. So, change the default sampling size of custom skills to "1.0", which will include all their generated skill data in the mixed output per skill leaf node instead of truncating it to 30 per skill leaf node.

**Comment by ktam3** [comment]
Based on discussion during today's Data team call, Ben will be closing this Issue

**Comment by bbrowning** [comment]
Yes - for more context, while we do default sampling to 30 today we'll be exposing a data mixing API that users can use to tweak those mixing recipes and adjust this from 30 to whatever value they want. In light of that, leaving this at some default number for now seems fine (and 30 is as good as any).
